### DISCUSSION TITLE
Fallout: Brotherhood of Steel

### DISCUSSION THREAD

**Discussion by dnewberry4** [JollyShmo/CSV_Import_Blender#4] [General]
Originally I tried emailing this to you, but unfortunately it bounced back, and this is really the only way I can think of to reach out at this point. This is specifically for JollyShmo, and is largely just the email I tried to send Originally.

I'm a member of the Fallout modding community, and I'm currently attempting to extract game objects from the Xbox game Fallout: Brotherhood of Steel. The only effective way I've found so far is using RenderDoc, but even then the way the game cuts up meshes into tris means reassembling anything close to useable for my modeling research is nigh impossible.

I've been able to use your Blender CSV program to make some kind of progress, but without any UV data and with only the vertices intact, since the generated tris just end up a mess.

Basically all of this to say that I really need your assistance to try and get something out of all this. I know a bit of python, but when it comes to Blender integration I'm basically a layman. 

If you're willing to look into this for me, I'd be very appreciative. Just let me know and I'll send you over the csv.

Hopefully you see this, and thanks for all the work you've already done on this.

**Comment by JollyShmo** [comment]
Hey @dnewberry4,
Sorry for the late response, I would be more than thrilled to take on a challenge like that. I also was working on a csv integration python script to bring multiple csv's together, but again I never tried that game. How many slices does it make? Do you have discord? you can email me here [codewizardjolly@protonmail.com](codewizardjolly@protonmail.com).

~ Jolly Joe

**Comment by JollyShmo** [comment]
> Originally I tried emailing this to you, but unfortunately it bounced back, and this is really the only way I can think of to reach out at this point. This is specifically for JollyShmo, and is largely just the email I tried to send Originally.
> 
> I'm a member of the Fallout modding community, and I'm currently attempting to extract game objects from the Xbox game Fallout: Brotherhood of Steel. The only effective way I've found so far is using RenderDoc, but even then the way the game cuts up meshes into tris means reassembling anything close to useable for my modeling research is nigh impossible.
> 
> I've been able to use your Blender CSV program to make some kind of progress, but without any UV data and with only the vertices intact, since the generated tris just end up a mess.
> 
> Basically all of this to say that I really need your assistance to try and get something out of all this. I know a bit of python, but when it comes to Blender integration I'm basically a layman.
> 
> If you're willing to look into this for me, I'd be very appreciative. Just let me know and I'll send you over the csv.
> 
> Hopefully you see this, and thanks for all the work you've already done on this.

I just added the combo tool [CSV Combo Info](https://github.com/JollyShmo/CSV_Import_Blender/wiki) I am working on that allows you to combine multiple csv files with python. Let me know if you want me to check out the file myself. Happy Coding.
